jbmc/src/java_bytecode/generic_parameter_specialization_map_keys.h

```diff
@@ -33,7 +33,7 @@ class generic_parameter_specialization_map_keyst
     for(const auto key : erase_keys)
     {
       PRECONDITION(generic_parameter_specialization_map.count(key) != 0);
-      (*generic_parameter_specialization_map.find(key)).second.pop();
+      (*generic_parameter_specialization_map.find(key)).second.pop_back();
       if((*generic_parameter_specialization_map.find(key)).second.empty())
```

Comment:
there are three successive lookups for the same key in the map, this could be reduced to one.